app/phone/src/main/java/dev/jdtech/jellyfin/fragments/SeasonFragment.kt

```diff
@@ -33,13 +43,45 @@ class SeasonFragment : Fragment() {
     private val args: SeasonFragmentArgs by navArgs()
 
     private lateinit var errorDialog: ErrorDialogFragment
+    private lateinit var downloadPreparingDialog: AlertDialog
 
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?,
     ): View {
         binding = FragmentSeasonBinding.inflate(inflater, container, false)
+        val menuHost: MenuHost = requireActivity()
+        menuHost.addMenuProvider(
+            object : MenuProvider {
+                override fun onCreateMenu(menu: Menu, menuInflater: MenuInflater) {
+                    menuInflater.inflate(dev.jdtech.jellyfin.core.R.menu.season_menu, menu)
+                }
+
+                override fun onMenuItemSelected(menuItem: MenuItem): Boolean {
+                    return when (menuItem.itemId) {
+                        dev.jdtech.jellyfin.core.R.id.action_download_season -> {
```

Comment:
Can change the resource IDs to use the same:

```suggestion
                        CoreR.id.action_download_season -> {
```